twisher/src/main/java/posts/ServletForPosts.java

```diff
@@ -0,0 +1,175 @@
+package posts;
+
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Stream;
+public class ServletForPosts extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String id = req.getParameter("id");
+        Post post = Posts.posts1.getPostByID(id);
+        String str;
+        if(post == null)
+            str = "Error";
+        else
+            str = post.toString();
+        resp.getOutputStream().write(str.getBytes(StandardCharsets.UTF_8));
+
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        if(req.getRequestURL().indexOf("/add") != -1) {
```

Comment:
(cosmetic) Please observe spaces and brackets.
Never skip brackets after `if (...)` and as well after `else`